manifest.json

```diff
@@ -237,6 +235,37 @@
             }
           }
         ]
+      },
+      "includeTwitterMetaTags": {
+        "oneOf": [
+          {
+            "properties": {
+              "includeTwitterMetaTags": {
+                "enum": [false]
+              }
+            }
+          },
+          {
+            "properties": {
+              "includeTwitterMetaTags": {
+                "enum": [true]
+              },
+              "twitterUsername": {
+                "title": "admin/store.twitterUsername.title",
+                "description": "admin/store.twitterUsername.description",
+                "type": "string"
+              },
+              "twitterCard": {
```

Comment:
This dropdown is not being rendered with title and description in the Site Editor for some reason... could you also look into that? It's just showing up as a dropdown with no context.

![Screen Shot 2021-09-10 at 14 09 42](https://user-images.githubusercontent.com/27777263/132891628-10f93811-8d4a-4699-b97f-79947bb51111.png)

Reply:
Hi @victorhmp 
I was doing a little digging to see what is causing the issue of the title and description not being displayed and the dropdown not having the correct style.
I noticed that we are using [react-jsonschema-form](https://www.npmjs.com/package/react-jsonschema-form) to construct the store settings (for the apps' settings as well). 
After reading the documentation, I don't see what we are doing wrong here. Maybe it's something when the VTEX Styleguide is being applied to the components.

After testing the same settings inside an apps settings form, this is what we get:

https://user-images.githubusercontent.com/17585823/133616291-7f74d07d-4059-405f-a8eb-0e8cf0c08e3a.mov



When testing the settings directly with react-jsonschema-form, this is what we get:

https://user-images.githubusercontent.com/17585823/133615266-159393f6-73cc-40a3-a6f7-033a034e7dcf.mov

As you can see, in both cases the title and the description is displayed and the dropbox is styled correctly.


What should we do?

Reply:
Awesome! Great investigation! I have an idea!
Could you try importing the `Dropdown` component from `react/components/form/Dropdown` and adding it to the `widgets` object created [here](https://github.com/vtex-apps/admin-pages/blob/master/react/components/admin/store/StoreForm/index.tsx#L43)? This might just do it!

Reply:
Hi @victorhmp 
I tried your approach but we are still getting the same results.

Reply:
Hi team!
Is there anything else we need to do about this app?